homeassistant/components/wittiot/sensor.py

```diff
@@ -0,0 +1,465 @@
+        sensor_type=MAIN_DATA,
+        name="Wind Speed",
+        icon="mdi:weather-windy",
+        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_WG,
+        sensor_type=MAIN_DATA,
+        name="Wind Gust",
+        icon="mdi:weather-windy",
+        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_DWM,
+        sensor_type=MAIN_DATA,
+        name="Day Wind Max",
+        icon="mdi:weather-windy",
+        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_UV,
+        sensor_type=MAIN_DATA,
+        name="UV-Index",
+        native_unit_of_measurement="Index",
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_SR,
+        sensor_type=MAIN_DATA,
+        name="Solar Irradiance",
+        native_unit_of_measurement=UnitOfIrradiance.WATTS_PER_SQUARE_METER,
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_RR,
+        sensor_type=MAIN_DATA,
+        name="Rain Rate",
+        icon="mdi:water",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_ER,
+        sensor_type=MAIN_DATA,
+        name="Rain Event",
+        icon="mdi:water",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_DR,
+        sensor_type=MAIN_DATA,
+        name="Rain Day",
+        icon="mdi:water",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_WR,
+        sensor_type=MAIN_DATA,
+        name="Rain Week",
+        icon="mdi:water",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_MR,
+        sensor_type=MAIN_DATA,
+        name="Rain Month",
+        icon="mdi:water",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key=TYPE_YR,
+        sensor_type=MAIN_DATA,
+        name="Rain Year",
+        icon="mdi:water",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
```

Comment:
Rain what? Rain this week, rain next week? Please define clearer names for the entities (as translation of course).

Also, are these aggregations? In that case it should use the state_class Total